tests/ops/test_hamiltonian.py

```diff
@@ -20,6 +20,13 @@
 import pennylane as qml
 from pennylane import numpy as pnp
```

Comment:
Sorry, this entire file is just cosmetics - I realised I can simplify the tests by parametrising with the `converter`